graphql-schemas/schema.yml

```diff
@@ -2273,6 +2273,20 @@
   - { name: query, type: string }
   - { name: queries, type: string, isList: true }
 
+- name: QueryValidationQueryItem_v1
```

Comment:
nit: would it make sense to detach the QueryItem from the validation context by just naming it QontractQuery? this way queries could be referenced in a type safe manner from other places as well

Reply:
renamed in https://github.com/app-sre/qontract-schemas/pull/153/commits/c9023097a9d40060159ede5130d493538bb89a5a